### PR TITLE
Globals cleanup

### PR DESCRIPTION
**Pull request by jmythms**
Pull request overview
---------------------
Minor cleanup of globals in the following files:

- AirflowNetworkBalanceManager.hh
- FluidProperties.hh
- PhotovoltaicThermalCollectors.hh
- RefrigeratedCase.hh
- SetPointManager.hh

- Small changes that include externs and statics in multiple files. 

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/AirflowNetworkBalanceManager.hh [hunk 1]
```diff
@@ -287,6 +287,14 @@ namespace AirflowNetworkBalanceManager {
         bool UpdateAirflowNetworkMyOneTimeFlag = true;
         bool UpdateAirflowNetworkMyOneTimeFlag1 = true;
 
+        // CalcAirflowNetworkAirBalance variables
+        int ErrCountVar = 0;
+        int ErrCountHighPre = 0;
+        int ErrCountLowPre = 0;
+        int ErrIndexHighPre = 0;
+        int ErrIndexVar = 0;
+        int ErrIndexLowPre = 0;
```
Comment (Myoldmopar) [inline src/EnergyPlus/AirflowNetworkBalanceManager.hh:296]: These have to be reset in the clear_state method.

File: src/EnergyPlus/FluidProperties.cc [hunk 2]
```diff
@@ -119,75 +119,31 @@ namespace FluidProperties {
...
-        RefrigData.deallocate();
-        RefrigErrorTracking.deallocate();
-        GlyRawData.deallocate();
-        GlycolData.deallocate();
-        GlycolErrorTracking.deallocate();
+void clear_state()
+{
 #ifdef EP_cache_GlycolSpecificHeat
-        cached_t_sh.deallocate();
+    cached_t_sh.deallocate();
```
Comment (Myoldmopar) [inline src/EnergyPlus/FluidProperties.cc]: Honestly, I'm thinking that we just don't need to clear this.  This is like one of the only things I can think of in E+ that we can simply leave in place.

**Review by Myoldmopar — COMMENTED** [review]
I think this needs a couple tweaks, but I'll make them locally later.  I think I want to wait on CI to do a pass over it.  At least the Github Action build which builds with the compiler flags swapped.

File: src/EnergyPlus/AirflowNetworkBalanceManager.hh (hunk 1 above)
Comment (jmythms) [reply src/EnergyPlus/AirflowNetworkBalanceManager.hh:296]: Ok, Thanks for pointing this out!

Comment (Myoldmopar) [reply src/EnergyPlus/AirflowNetworkBalanceManager.hh:296]: No problem!  And they are conflicted as I had addressed this in my static pass, so I'll just clean up the conflicts and we'll move on.

**Comment by Myoldmopar** [comment]
OK, this one next, I'll resolve conflicts and get it tested locally.

**Comment by jmythms** [comment]
Thank you!

**Comment by Myoldmopar** [comment]
Quick checked this branch with develop again and your latest commit and all is still well.  I'm pushing the resolved branch and then merging.  Thanks @jmythms !

**Comment by jmythms** [comment]
Thank you very much!

**Pull request merged**